content/en/docs/contributing/docs/style-guide.md

```diff
@@ -0,0 +1,400 @@
+---
+title: Documentation Style Guide
+linkTitle: Documentation Style Guide
+weight: 20
+description: >
+    Style Guide for Flux Documentation
+---
+## Documentation formatting standards
+
+This page gives writing style guidelines for the Flux documentation. These are guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+### Use upper camel case for API objects
+
+Do | Don't
+:--| :-----
+| The GitRepository resource is responsible for ... | The Git Repository resource is responsible for ... |
```

Comment:
Can we suggest people put a code span around types, as well? E.g., `GitRepository`. Or is this considered distracting (if so please tell me, I do it all the time!)

Reply:
I've added a bit more of a blurb here lmk what you think

Reply:
It occurs to me that you could combine this advice with "use code blocks", either by moving that item here, or moving this later. But if that doesn't work, the caveat you've given is fine :+1: